contracts/src/instances/eeth/EETHConversions.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IERC20 } from "../../interfaces/IERC20.sol";
+import { ILiquidityPool } from "etherfi/src/interfaces/ILiquidityPool.sol";
+import { FixedPointMath, ONE } from "../../libraries/FixedPointMath.sol";
+import { IeETH } from "etherfi/src/interfaces/IeETH.sol";
```

Comment:
I think `IEETH` would be more consistent with our other yield sources. For example, we have `IEzETH`

Reply:
that's the etherfi source code not an interface I created